### PR TITLE
Create WEEKLY_MEETINGS.md

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: WEEKLY_MEETINGS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+We've agreed that we'll have a weekly meeting, every Thursday at 09:00. Not all the members have to be present. If we expect everybody to be there, scheduling can get really challenging, and this won't get easier with time, it will only get more difficult as we grow. Also, is important to keep doing it continiously so into becomes part of our routine.
```
Comment (marijnschilling) [inline WEEKLY_MEETINGS.md]: Also, it is important to keep doing it continuously so it becomes part of our routine.

**Comment by elland** [comment]
@3lvis due to my current schedule, I don't think I can make it on Thu 9am. Can we make it after lunch perhaps?

**Comment by 3lvis** [comment]
@Elland Which day you can make it at 9 am?

**Comment by elland** [comment]
@3lvis I'd have to see next week. Doesn't look good at the moment. 10am perhaps.

**Comment by marijnschilling** [comment]
I can't make 9 am on tuesdays

**Comment by 3lvis** [comment]
@Elland you can't any day at 09:00?

**Comment by elland** [comment]
@3lvis consistently? Hard to tell at the moment.

**Pull request merged**